packages/services/f-performance/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@justeat/f-performance",
+  "description": "Fozzie Performance - RUM performance metric collector",
+  "version": "0.1.0",
+  "main": "dist/f-performance.umd.min.js",
+  "files": [
+    "dist"
+  ],
+  "homepage": "https://github.com/justeat/fozzie-components/tree/master/packages/services/f-performance",
+  "contributors": [
+    "Github contributors <https://github.com/justeat/fozzie-components/graphs/contributors>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:justeat/fozzie-components.git"
+  },
+  "bugs": {
+    "url": "https://github.com/justeat/fozzie-components/issues"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "keywords": [
+    "fozzie"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn lint && yarn test && yarn build",
+    "lint": "vue-cli-service lint",
```

Comment:
Might be more of a thing for the generator project but maybe we should consider offering a Vite version of the npm packages and tasks, as plain JS projects don't really need Vue-CLI to handle things like building, linting etc - I'm thinking of projects like `f-consumer-oidc`, `f-http`, etc.

Reply:
^ yeah agreed, basically none of our services use Vue CLI anymore

Reply:
@tommcclean Are you ok to create an EE ticket for this? I'm not sure what needs updating to put the service build on Vite myself (can pull this in then in the next couple of weeks)